mlops/bicep/sandbox_fl_eyesoff_gpu.bicep

```diff
@@ -0,0 +1,50 @@
+// This BICEP script will fully provision a federated learning sandbox
+// with eyes-off orchestrator and silos storages
+// and only one compute (cpu by default)
+
+targetScope = 'resourceGroup'
+
+// please specify the base name for all resources
+@description('Base name of the demo, used for creating all resources as prefix')
+param demoBaseName string = 'fldemo'
+
+@description('Region of the orchestrator (workspace, central storage and compute).')
+param orchestratorRegion string = resourceGroup().location
+
+@description('List of each region in which to create an internal silo.')
+param siloRegions array = [
+  'australiaeast'
+  'eastus'
+  'westeurope'
+]
+
+@description('The VM used for creating compute clusters in orchestrator and silos.')
+param computeSKU string = 'Standard_NC6'
+
+@description('Uses public network access for the orchestrator storage, allowing it to be eyes-on.')
+param orchestratorEyesOn bool = true
```

Comment:
should we keep it `false` by default? (in all eyes-off scripts)

Reply:
that's a good question, I initially did set false by default, then I thought maybe the users would like to actually see just the orchestrator.

Reply:
I switched to false by default.